media/css/pebbles/includes/lib.less

```diff
@@ -243,12 +311,8 @@
 }
 
 
-
-/****************************************************************************/
-// Classes used to hide content from users. Choose wisely!
-
-// Hide visually, but not to screen readers
-.visually-hidden() {
+// Hide an element visually, but not from screen readers
+.a11y-hidden() {
```

Comment:
Could the name `a11y-hidden` sound a bit confusing, in that it could be interpreted as an element being hidden from screen readers? (whereas it is actually the reverse).